airflow/contrib/hooks/gcp_video_intelligence_hook.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from google.cloud.videointelligence_v1 import VideoIntelligenceServiceClient
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+class CloudVideoIntelligenceHook(GoogleCloudBaseHook):
+    """
+    Hook for Google Cloud Video Intelligence APIs.
+    """
+
+    _conn = None
+
+    def __init__(self, gcp_conn_id="google_cloud_default", delegate_to=None):
+        """
+        :param gcp_conn_id: The connection ID to use when fetching connection info.
+        :type gcp_conn_id: str
+        :param delegate_to: The account to impersonate, if any.
+            For this to work, the service account making the request must have
+            domain-wide delegation enabled.
+        :type delegate_to: str
+        """
+        super(CloudVideoIntelligenceHook, self).__init__(gcp_conn_id, delegate_to)
+
+    def get_conn(self):
+        """
+        :rtype: google.cloud.videointelligence_v1.VideoIntelligenceServiceClient
+        """
+        if not self._conn:
+            self._conn = VideoIntelligenceServiceClient(credentials=self._get_credentials())
+        return self._conn
+
+    def annotate_video(
+        self,
+        input_uri=None,
+        input_content=None,
+        features=None,
+        video_context=None,
+        output_uri=None,
+        location=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        Performs video annotation.
+
+        :param input_uri: Input video location. Currently, only Google Cloud Storage URIs are supported,
+            which must be specified in the following format: gs://bucket-id/object-id.
+        :type input_uri: str
+        :param input_content: The video data bytes.
+            If unset, the input video(s) should be specified via input_uri. If set, input_uri should be unset.
+        :type input_content: bytes
+        :param features: Requested video annotation features.
+        :type features: list[Feature]
```

Comment:
Can you provide a full class path?

Reply:
There is no classpath in offcial documentation, and Feature is really just an enum

Reply:
Changed to `list[google.cloud.videointelligence_v1.VideoIntelligenceServiceClient.enums.Feature]`

Reply:
I created a issue in google-cloud-python library https://github.com/googleapis/google-cloud-python/issues/7503